skaffold.yaml

```diff
@@ -0,0 +1,156 @@
+apiVersion: skaffold/v2alpha2
```

Comment:
@donnerc This is the most important file, with the not-yet-existing `gitlab-ci.yaml` file. In this file, we configure how we build and deploy our software. I've started defined a few different profiles which are currently incomplete (`staging` and `production`). The most important stage here today is the default stage, the one not attached to any profile, as it is what we use in our local development k8s clusters.